### ISSUE TITLE
🙃

### ISSUE THREAD

**Issue by enoleriand-alt** [saydog/saydog-framework#1]
Can your tool run in a Linux environment?

**Comment by s1l3nt78** [comment]
Yes this tool can run in linux. Just change 'python __main__.py' in the run script to 'python3.7 __main__.py'

**Comment by enoleriand-alt** [comment]
Your project is good and suitable for working with children

**Comment by enoleriand-alt** [comment]
🤣😆

**Comment by s1l3nt78** [comment]
This isnt my project, i just know how to get it working. Though it doesnt contain any explicit content if that is your worry.
